From 9a57781d978be61289afd8ae99e3aeb5c36eb208 Mon Sep 17 00:00:00 2001
From: John Johansen <john.johansen@canonical.com>
Date: Thu, 14 Mar 2019 13:34:14 -0700
Subject: [PATCH 08/31] UBUNTU: SAUCE: Revert "apparmor: Fix warning about
 unused function apparmor_ipv6_postroute"

Revert currently unused by Ubuntu apparmor features that require secid
support, to enable apparmor LSM stacking with the 5.1 LSM stacking
patchset.

This reverts commit a1a02062ad466052a34a8c4323143ccf9726eb52.

Signed-off-by: John Johansen <john.johansen@canonical.com>
Signed-off-by: Seth Forshee <seth.forshee@canonical.com>
---
 security/apparmor/lsm.c | 2 --
 1 file changed, 2 deletions(-)

diff --git a/security/apparmor/lsm.c b/security/apparmor/lsm.c
index e0e993e8533c..de90d6be5137 100644
--- a/security/apparmor/lsm.c
+++ b/security/apparmor/lsm.c
@@ -1743,14 +1743,12 @@ static unsigned int apparmor_ipv4_postroute(void *priv,
 	return apparmor_ip_postroute(priv, skb, state);
 }
 
-#if IS_ENABLED(CONFIG_IPV6)
 static unsigned int apparmor_ipv6_postroute(void *priv,
 					    struct sk_buff *skb,
 					    const struct nf_hook_state *state)
 {
 	return apparmor_ip_postroute(priv, skb, state);
 }
-#endif
 
 static const struct nf_hook_ops apparmor_nf_ops[] = {
 	{
-- 
2.25.1

